_studio/mfx_lib/encode_hw/h265/include/mfx_h265_encode_vaapi.h

```diff
@@ -46,6 +48,19 @@ do {                                               \
     }                                              \
 } while (0)
 
```

Comment:
If we replace MFX_DESTROY_VABUFFER to new function and we will not use this define anymore - I suggest to delete MFX_DESTROY_VABUFFER